pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -2719,6 +2713,26 @@ def finalize_model(
             estimator=estimator, fit_kwargs=fit_kwargs, model_only=model_only,
         )
 
+    def ts_tests(
```

Comment:
I suggest hear to change the name to `ts_stat_tests` or something that makes reference to calculating an statistical test, otherwise the name is not clear about what type of time series tests is running.